recipes/norm/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+import yaml
+
+
+class NormConan(ConanFile):
+    name = "norm"
+    description = "A reliable multicast transport protocol"
+    topics = ("conan", "norm", "multicast", "transport protocol")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.nrl.navy.mil/itd/ncs/products/norm"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    license = "NRL"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["NORM_CUSTOM_PROTOLIB_VERSION"] = "./protolib"
+        self._cmake.configure()
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        if self.options.shared:
+            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*proto*")
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "norm"
+        self.cpp_info.names["cmake_find_package_multi"] = "norm"
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
```suggestion
        self.cpp_info.libs = ["norm", "protokit"]
```

Don't rely on `tools.collect_libs()` when there is more than 1 lib with inter dependencies. norm depends on protokit so order matters and we should be explicit.

Reply:
Maybe protokit is always built as a static lib and not installed if norm is shared.